yazi-plugin/preset/components/folder.lua

```diff
@@ -21,6 +21,12 @@ function Folder:linemode(area, files)
 			spans[#spans + 1] = ui.Span(time and os.date("%y-%m-%d %H:%M", time // 1) or "")
 		elseif mode == "permissions" then
 			spans[#spans + 1] = ui.Span(f.cha:permissions() or "")
+		elseif mode == "owner" then
+			local username = ya.user_name(f.cha.uid)
```

Comment:
`ya.user_name`, `ya.group_name`, `cha.uid`, and `cha.gid` are Unix-specific attributes/methods, they will break the UI rendering on the Windows platform.

Perhaps we can expose them to the Windows platform, but just return a `nil` value? Or add this code snippet to the [Tips page](https://yazi-rs.github.io/docs/tips) so Unix users can copy it from there? Curious your thoughts!

Reply:
I think it is pretty important on Unix to be able to see the user and group of files by default. Additionally this functionality somewhat [already exists](https://yazi-rs.github.io/docs/tips/#user-group-in-status) through the status bar.

It would be nice if this functionality could work on Windows as well. It looks doable [here](https://users.rust-lang.org/t/how-can-i-get-the-owner-of-a-file-on-windows/47563), but more complicated than it should be.

If we expose these methods to the Windows platform we should remove the ability to see the ownership key-bind in the help page. It would be confusing for a Windows user to see the option and have it do nothing.

I am happy to implement either safely hiding it on Windows or having the ownership functionality on Windows, up to you.

Reply:
Great, please implement the ownership functionality on Windows!